package-lock.json

```diff
@@ -0,0 +1,945 @@
+{
```

Comment:
How come we haven't had a `package-lock.json` until now?

Reply:
Whoops, must've used npm instead of yarn at some point, deleted the file